src/lib/config.ts

```diff
@@ -1171,6 +1171,16 @@ export class default_config {
      */
     historyresults = 50
 
+    /**
+     * Reduce the size (line count) of ex command history if the size is larger than the historycmdcountalert when tridactyl started. The size will be reduce to the historycmdcountlimit.
+     */
+    historycmdcountalert = 768
+
+    /**
+     * Reduce the size (line count) of ex command history if the size is larger than the historycmdcountalert when tridactyl started. The size will be reduce to the historycmdcountlimit.
+     */
+    historycmdcountlimit = 512
```

Comment:
I'd rather have one setting rather than two. Maybe remove 1/3 when it hits the limit.

512 seems pretty small, I'd set it to something much bigger like 100k. I think most of the time people are searching through it rather than just scrolling up